include/highfive/H5Object.hpp

```diff
@@ -9,6 +9,8 @@
 #ifndef H5OBJECT_HPP
 #define H5OBJECT_HPP
 
+#include "H5Exception.hpp"
```

Comment:
Any reason why `H5Exception.hpp` isn't part of the include section below. Besides standard includes should be a in [separated sections](https://github.com/BlueBrain/hpc-coding-conventions/tree/master/cpp/formatting#sort-includes-in-different-sections).